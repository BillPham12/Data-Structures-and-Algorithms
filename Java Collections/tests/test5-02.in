1
123
hello
mey i have
123
1
my name is g
21d312d3
12d312
d3
12d3
12d
312
d312
d3
123d
12
312
312
3
123
123
123
12
312
312
3
123
12
312
312
312
312312312312312
osu