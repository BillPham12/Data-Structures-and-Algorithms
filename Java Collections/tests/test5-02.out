1
12
123
123d
12d
12d3
12d312
21d312d3
3
312
312312312312312
d3
d312
hello
mey i have
my name is g
osu
